Set --source and --target when available.

tmpfs opened the pull request:
From the `ANDROID_SOURCE_VERSION` and `ANDROID_RELEASE_VERSION` environment variables.

Closes #1.

kevinaboos commented on src/env_paths/mod.rs:
```diff
@@ -1,25 +1,31 @@
-pub const ANDROID_JAR:                  &str = "ANDROID_JAR";
-pub const JAVA_HOME:                    &str = "JAVA_HOME";
-
+pub const ANDROID_SOURCE_VERSION: &str = "ANDROID_SOURCE_VERSION";
+pub const ANDROID_TARGET_VERSION: &str = "ANDROID_TARGET_VERSION";
```
kindly follow our existing indentation/alignment scheme here, thanks

kevinaboos reviewed (changes requested):
Thanks, this looks good to me. 

Can you undo all of the unrelated changes? I assume you ran a code formatter like rustfmt, but for this PR let's just restrict the changes to those relevant for `--source` and `--target`. Thanks!

Also, please add a description of the two new env vars to the crate-level documentation such that they appear [here](https://docs.rs/android-build/latest/android_build/#environment-variables-in-use)

tmpfs committed:
Set --source and --target when available.

From the `ANDROID_SOURCE_VERSION` and `ANDROID_RELEASE_VERSION` environment variables.

tmpfs commented:
Ah yeh, sorry about that @kevinaboos - my neovim config does that by default. I switched to vim and have addressed all your points and force pushed, I think it follows the convention now.

The diff is much clearer now 👍

kevinaboos commented:
Thanks, looks good now! One more request -- can you give some examples (in the docs) of what sort of values/formats are supported for the new `--source`/`--target` arguments? I'm not familiar with them, but I assume it's something like "--source 1.7". 

(Note that in the crate-level docs I gave similar examples for android versions, since they can be confusing).

kevinaboos commented:
Ah, actually, a couple more comments:

1. Are these source and target versions related to Android at all? They seem to be exclusively related to _Java_, not Android per se. If that's the case, we should name them accordingly: `JAVA_SOURCE_VERSION` and `JAVA_TARGET_VERSION`
2. Is it "target" or "release"? In your PR description, you wrote "ANDROID_RELEASE_VERSION" but I assume you mean "target".

tmpfs commented:
Ah, good points. Yes it's actually more Java related so I will rename them. Will add examples too, should be something like "17". I do find the Java versioning crazy confusing though 😆

tmpfs committed:
Prefer JAVA env variable prefix.

tmpfs commented:
Correct @kevinaboos, using `RELEASE` in the issue was a mistake we should use SOURCE and TARGET so it matches the `javac` options.

I've switched to the `JAVA` prefix for the variables and clarified in the docs to use `17` which is the language version (corresponding to JDK 1.7 I think), do you think this is ok now?

kevinaboos reviewed (approved):
yep, looks good now. Thanks for the contributions!
